### DISCUSSION TITLE
请问smb具体如何使用？

### DISCUSSION THREAD

**Discussion by GHOSTOPOST** [AlistGo/alist#2215] [Q&A]
3.3.0增加了smb，具体如何使用呢？

**Comment by yefengwu** [comment]


**Reply by greatzkk** [reply]
是windows的Alist吗？  我这边群晖上docker挂smb死活挂不上

**Comment by Sandman126** [comment]


**Reply by yefengwu** [reply]
不好意思，我理解错了，我的是挂载smb到alist。挂载alist smb到win还没试过。

**Reply by SheltonZhu** [reply]
现在不支持alist 挂到win吧？
只有webDAV做到双向了，而且是因为都用的http协议，开一个web服务就够了。
如果要搞smb还要再开个服务监听端口哦

**Reply by wfy9041** [reply]
这个有用

**Comment by ChenC2022** [comment]
Alist v3 的SMB这样设置。
如果只是将共享名直接作根，那么在Root folder path 项中留个点"."
另外，如果是Docker 安装的视Docker的情况可能需要单独装个smbclient程序。
![image](https://user-images.githubusercontent.com/98141797/201004845-19b7b316-44ff-4efe-b112-e5841a8bfce0.png)

**Reply by miaoermua** [reply]
感谢，看到有 SMB 选项不知道怎么用文档也没有，用了这个方法挂载上了

**Reply by GuoliangLai** [reply]
感谢，能挂载上，但是提示failed get objs: failed to list objs: open .xxx: file does not exist，不知道是啥问题，是群晖开的smb服务

**Reply by glenman** [reply]
亲自测试了一下可行，将Alist来分享我路由器上的挂载的硬盘盒，就是必须要准确定位到具体的某个文件夹上，比如SMB共享的是这样的\\192.168.8.1\Media_Lib\TV-Series, 那对应的配置中地址为：192.168.8.1:445, 共享名就是Media_lib, 要共享的文件夹名称就是TV_Series

**Reply by dyzzc** [reply]
> 感谢，能挂载上，但是提示failed get objs: failed to list objs: open .xxx: file does not exist，不知道是啥问题，是群晖开的smb服务

试试把“根文件夹路径”里的内容不填或是恢复初始的“.”

**Reply by xxxxha** [reply]
我这测试，“根文件夹路径”，要以‘./’ 开头，才不会出错。比如填'.'，或者‘./Movie’

**Reply by magicvdog** [reply]
> > 感谢，能挂载上，但是提示failed get objs: failed to list objs: open .xxx: file does not exist，不知道是啥问题，是群晖开的smb服务
> 
> 试试把“根文件夹路径”里的内容不填或是恢复初始的“.”
按你说的解决了问题

**Reply by asong2022** [reply]
谢谢！！！很大帮助！！

**Reply by curryhendry** [reply]
您好，请问我这个提示是咋回事，怎么处理比较合适？感谢！

connection error: read tcp 192.168.31.114:12077->192.168.31.114:445: read: connection reset by peer

**Reply by curryhendry** [reply]
曲线救国了，我在R3D上也安装了一个alist的，互相添加了Alis V3从而实现了相互套娃。。

**Reply by lyc0gh** [reply]
> > > 谢谢，能挂载上，但是提示获取 objs 失败：无法列出 objs：打开 .xxx：文件不存在，不知道是啥问题，是群晖开的 smb 服务
> > 
> > 
> > 试试把“根文件夹路径”里的内容填入或者恢复初始的“。”
> > 按你所遇到的问题

太感谢了！高手！

**Reply by biyeahli** [reply]
> Alist v3 的SMB这样设置。 如果只是将共享名直接作根，那么在Root folder path 项中留个点"." 另外，如果是Docker 安装的视Docker的情况可能需要单独装个smbclient程序。 ![image](https://user-images.githubusercontent.com/98141797/201004845-19b7b316-44ff-4efe-b112-e5841a8bfce0.png)

为啥我挂载后，还是失败，共享的文件没有权限访问，这是为什么啊

**Reply by tjchocolate** [reply]
failed get objs: failed to list objs: connection error: invalid transport format
添加smb的时候报这个警怎么解决

**Reply by reteng** [reply]
Mark

**Comment by MilkTeaNo7** [comment]
你还能挂SMB，我连SMB这个选项都没了

**Comment by GHOSTOPOST** [comment]
原来是把smb挂载到alist，而不是alist通过smb共享出来。我以为这个smb能像webdav共享出来。失望了。

**Reply by wangrui1573** [reply]
原来是把smb挂载到alist，而不是alist通过smb共享出来。。

**Reply by ChenC2022** [reply]
不需要以”smb能像webdav共享出来“，windows 的文件管理器可以直接挂载webdav，当然也可以用RaiDrive这类的工具。

**Reply by zjy2931** [reply]
这是汉语吗 咋看不懂呢   挂在上去和共享出来啥区别  意思是smb只能在局域网访问 ？

**Reply by NXY666** [reply]
> 这是汉语吗 咋看不懂呢   挂在上去和共享出来啥区别  意思是smb只能在局域网访问 ？

意思是能使用smb协议访问alist，而“挂上去”指的是添加一个smb的存储。

**Comment by libixi123** [comment]
nas中doker部署alist，只可以挂载根（即上文提到的--将共享名直接作根，那么在Root folder path 项中留个点"."）。如果共享次一级目录，提示failed get objs: failed to list objs: open \: leading '\' is not allowed in。

**Reply by denki-san** [reply]
这很有帮助👍
./ 只能挂载根目录下的一级文件夹，而无法访问一级文件夹下面的文件
. 是挂载根目录以及下面所有的文件

**Comment by giceli** [comment]
我成功了一半，可能以打开。但是在打开子目录的时候 出现了错误:
failed get objs: failed to list objs: open Public: response error: The contacted server does not support the indicated part of the DFS namespace.
可能是因为我挂载的这个是CIFS吧。

**Comment by wang3150315** [comment]
![image](https://user-images.githubusercontent.com/33510782/226152808-ca79da08-f70e-4476-ac83-85bcc7233fb2.png)
我这个为啥挂不上啊

**Reply by wang3150315** [reply]
Failed init storage: response error: The attempted logon is invalid. This is either due to a bad username or authentication information.
这个提示

**Reply by dyzzc** [reply]
你的地址是//192.168.50.86/Desktop-s3s7f9s 吗？
你可以试试跟文件夹路径输入“.”，分享名称输入“Desktop-s3s7f9s”
按照我的测试，只能挂载smb的根目录，你这个“影音文件”应该挂不上

**Reply by wang3150315** [reply]
> 你的地址是//192.168.50.86/Desktop-s3s7f9s 吗？ 你可以试试跟文件夹路径输入“.”，分享名称输入“Desktop-s3s7f9s” 按照我的测试，只能挂载smb的根目录，你这个“影音文件”应该挂不上

<img width="655" alt="image" src="https://user-images.githubusercontent.com/33510782/226393558-d687a7a2-b7e5-478a-873f-f31bab2cb835.png">

还是不行,这个用户名我应该输入什么呢 我用的不是本地的windows 账号

**Reply by wang3150315** [reply]
> 你的地址是//192.168.50.86/Desktop-s3s7f9s 吗？ 你可以试试跟文件夹路径输入“.”，分享名称输入“Desktop-s3s7f9s” 按照我的测试，只能挂载smb的根目录，你这个“影音文件”应该挂不上

Failed init storage: response error: The attempted logon is invalid. This is either due to a bad username or authentication information.
我现在变成这个提示了

**Reply by BrandonStudio** [reply]
你的用户名显然是有问题的。
看你的高级共享设置，如果勾选了“密码保护的共享”，那么需要填写有效的目标计算机账户的用户名和密码。如果没有勾选，用户名可以写Guest

**Comment by xxxxha** [comment]
我这测试，根文件夹路径，要以‘./’ 开头，才不会出错。比如填'.'，或者‘./Movie’

**Comment by yingmuzeng** [comment]
我挂载成功了。给大家分享一下。挂载的是局域网另一台windows电脑里的D盘，
![image](https://user-images.githubusercontent.com/128979225/227784419-1d0a8a80-04fc-40bb-a54e-64c883ca87fa.png)
根文件夹路径就是一个“.”,然后地址就填这台电脑的ip:445。用户名（必须得本地账户，不能是Microsoft账户）和密码就是登录用户名和密码。分项名称就填d。

**Reply by Terry-Wing** [reply]
的确，可以了

**Reply by ioioRhys** [reply]
这个是可以的

**Reply by maxoo1001** [reply]
域控的账户可以吗？有人试过吗

**Reply by londbell** [reply]
> 我挂载成功了。给大家分享一下。挂载的是局域网另一台windows电脑里的D盘， ![image](https://user-images.githubusercontent.com/128979225/227784419-1d0a8a80-04fc-40bb-a54e-64c883ca87fa.png) 根文件夹路径就是一个“.”,然后地址就填这台电脑的ip:445。用户名（必须得本地账户，不能是Microsoft账户）和密码就是登录用户名和密码。分项名称就填d。

有效，好用

**Reply by leochan2017** [reply]
Mark，我也成功了

**Comment by ccsmile8609** [comment]
我挂载成功了，能看到文件，但是无法下载，提示{"code":401,"message":"expire missing","data":null}，这是啥情况

**Comment by karasfjm** [comment]
挂载成功了，也能看到文件，但是都不能在线播放是怎么回事？

**Comment by a447948814** [comment]
我告诉你们,很简单，驱动是SMB。挂载路径就是叫啥名字/MUSIC。webdav策略就是本地代理。剩下的就比较重要：
地址直接是电脑ip+端口，比如：10.10.1.2:445，用户名密码就是电脑的管理员用户密码即可。分享名就是共享文件夹的名称，。OK搞定。

**Reply by a447948814** [reply]
注意。根文件夹路径下面有个小点  .   [可别丢了，不然共享文件夹下面的文件夹打不开。、

**Reply by a447948814** [reply]
上面那个yingmuzeng说的也对

**Reply by yy915cn** [reply]
> 我告诉你们,很简单，驱动是SMB。挂载路径就是叫啥名字/MUSIC。webdav策略就是本地代理。剩下的就比较重要： 地址直接是电脑ip+端口，比如：10.10.1.2:445，用户名密码就是电脑的管理员用户密码即可。分享名就是共享文件夹的名称，。OK搞定。

挂载共享下的文件夹是可以，但是怎么挂载共享的根目录呢

**Comment by DioBakery** [comment]
<img width="195" alt="image" src="https://github.com/alist-org/alist/assets/69229972/72f50e89-4935-4b41-a774-cd5ac56b48a7">
无论怎么填，都提示Failed init storage: response error: An invalid parameter was passed to a service or function.

**Reply by SyberRabbit** [reply]
哎，我也是

**Reply by NXY666** [reply]
我也是小米路由器，提示这个

**Reply by curryhendry** [reply]
曲线救国了，在小米路由上也安装了alist，然后挂alistv3可以读写。

**Reply by mengtaixiao** [reply]
我的京东云路由器鲁班，是无密码的SMB，也不能设置用户和密码，在alist输入用户名anonymous密码留空或者随便填，能挂载成功。
我也有小米路由器r3g，他的SMB是无密码的，也不能设置用户和密码，怎么都登不上，试过guest Guest anonymous nobody都不行，

有大佬试出来了踢我一脚

**Reply by onetosan** [reply]
把“启用签名”关闭看看

**Reply by winic57** [reply]
核心问题分析

您在Alist中挂载失败，收到的错误是 "response error: An invalid parameter was passed to a service or function."。这个错误通常是客户端（Alist）和服务器端（Linux Samba）之间的认证协议或参数不匹配导致的。

再结合您的smb.conf配置文件，根本原因就找到了：

在您的 smb.conf 的 [global] 部分，您使用了 security = share。

security = share：这是一个非常古老且已被官方废弃的安全模式。它不基于用户名进行认证，而是可以为整个共享目录设置一个密码。现代的SMB客户端（包括Alist底层使用的库）大多已经不再支持或默认不使用这种模式，它们期望的是基于用户的认证。

security = user：这是当前所有系统的标准和默认模式。客户端必须提供一个在Samba服务器上存在的用户名和对应的密码才能访问。

Alist尝试使用用户名 admin 和密码去连接，但您的Samba服务器配置为share模式，根本不理会这个用户名，导致认证参数协商失败，从而返回了 "invalid parameter" 的错误。

**Comment by ironccc** [comment]
我用truenas上的alist去挂一个局域网里的黑裙的smb，怎么都挂不上，之后把报错发出来给大佬们看一下

**Reply by ironccc** [reply]
![image](https://github.com/alist-org/alist/assets/33080565/9bcf2595-5b8b-46c4-91af-bf5ab6f5933b)

**Reply by CJY-123456** [reply]
挂载上了，备份群晖到本地硬盘，就是速度不太行，用的是2.5g网卡+2.5g交换机+ssd。
<img width="2185" height="1090" alt="image" src="https://github.com/user-attachments/assets/5d554dd6-5d96-45c8-a6db-685001c467de" />

**Comment by ChenC2022** [comment]
给你是设置示例你应该就明白了。

**Reply by ChenC2022** [reply]
举例：Windows 里看SMB的地址是这样的
![image](https://github.com/alist-org/alist/assets/98141797/75532c7b-4074-4153-bd0a-6ec92c99fb5a)
Alist 里挂载的设置是这样的
![image](https://github.com/alist-org/alist/assets/98141797/c2011401-9214-4cf1-8106-87afc0e189ac)
挂载后是这样的
![image](https://github.com/alist-org/alist/assets/98141797/9758d536-672b-432e-bd8d-e93b3c92e551)

**Reply by ChenC2022** [reply]
还看到上面几个关于挂载SMB一级根的设置问题。我也贴个示例图。
![image](https://github.com/alist-org/alist/assets/98141797/551f1b18-9631-4907-a1b5-37162a1b8f01)

![image](https://github.com/alist-org/alist/assets/98141797/a77f99ce-9dd0-49b7-a52a-55597ddb561f)
![image](https://github.com/alist-org/alist/assets/98141797/0ad72f20-4b79-4576-b59d-f77c0b631151)

**Reply by ironccc** [reply]
![image](https://github.com/alist-org/alist/assets/33080565/a51b8e2d-b1a5-40a4-a300-381b182b6b7f)
![image](https://github.com/alist-org/alist/assets/33080565/09c5c5dd-8736-4516-9271-d2ca978fa188)
我测试的路径和设置是这样的。。。还是不行

**Reply by yy915cn** [reply]
我这样设置，还是只会挂载Software那个文件夹

**Reply by PingZi-Wing** [reply]
我挂路由器的SMB，在填IP的时候傻逼的填了本机而不是路由器的😂

**Comment by ironccc** [comment]
还有一两个跟alist相关的其他问题一并上报一下吧：
1、从百度云和阿里云拉取同步云端的数据总是失败，上传没问题，报错截图之后传上来
2、阿里云open总是时不时地报token过期，进设置界面重新点一下保存 就好了，不知道是什么原因

**Comment by ranapankj** [comment]
![1111](https://github.com/alist-org/alist/assets/30015771/d1159d72-5945-4289-b016-9cb8a117552e)
![2](https://github.com/alist-org/alist/assets/30015771/c4dac58e-9888-458e-97e5-0ad72fbd53d2)

**Comment by yy915cn** [comment]
![image](https://github.com/alist-org/alist/assets/39397983/9e893ea6-09f7-41ce-a44d-e86686b04e3c)
想挂载整个根目录要怎么设置，记得以前v2直接可以挂

**Reply by dada8686** [reply]
我也想直接挂载根目录貌似是不行的，windows都不支持直接挂载根目录

**Comment by kokojacket** [comment]
挂在是能挂在上，但是完全无法显示缩略图，非常难受

**Comment by kexue-z** [comment]
![image](https://github.com/alist-org/alist/assets/71873002/34fdcdc2-9d7e-4888-8c5f-ad4643d13087)

如何挂载Everyone账号，目前用了 Everyone guest 大小写均不可用。

**Comment by lvxuan263** [comment]
alist 使用的 `https://github.com/hirochachacha/go-smb2` smb-client 库目前未支持 anonymous 访问，暂时只能使用账户登录。

`https://github.com/LeakIX/go-smb2` fork 自 hirochachacha 项目，替换了NTLM lib为 NTLMCPP，支持了匿名登录。有能力的同学可以自行切换go smb库~

**Reply by zgcj98** [reply]
![Uploading image.png…]()

**Comment by abc123sm** [comment]
群晖，给各位参考
189是我群晖的地址
不要填BlackQH这个机器名，直接填下级分享资料夹的名字
![image](https://github.com/alist-org/alist/assets/7129672/4386dfaf-181a-4a15-b3d7-e095ca59b52f)

**Reply by chrisuhg** [reply]
我跟你一样这样填，提示找不到，但是我电脑啥的设备都能正常挂载
<img width="523" alt="image" src="https://github.com/alist-org/alist/assets/34882677/9cf6df7c-dbb2-416f-82bf-683f2e042541">

**Reply by zjcnm** [reply]
> 群晖，给各位参考 189是我群晖的地址 不要填BlackQH这个机器名，直接填下级分享资料夹的名字 ![image](https://private-user-images.githubusercontent.com/7129672/301628196-4386dfaf-181a-4a15-b3d7-e095ca59b52f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDUyNDg5NDUsIm5iZiI6MTc0NTI0ODY0NSwicGF0aCI6Ii83MTI5NjcyLzMwMTYyODE5Ni00Mzg2ZGZhZi0xODFhLTRhMTUtYjNkNy1lMDk1Y2E1OWI1MmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDQyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA0MjFUMTUxNzI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjI4MTA5Yjk5MGRlNDJiOGJhMTIxMzM1NTQzODUwZGY0NzYxMDc2MjkxM2Q0M2EyNjA1OGZkMDkyMDczMWIxNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.zzTzJ_mqu8vydsp8dtC0Tbveb2OuZ2SOJm3vOp8Heas)

感谢  我的绿联NAS终于挂载上去了！！！

**Reply by fdxwy6** [reply]
有用且清晰，感谢！

**Comment by SyberRabbit** [comment]
有人能帮忙看看吗，怎么填都不行<img width="1280" alt="af690dce5ded15f6a8c1dac7d2ad2bc0" src="https://github.com/alist-org/alist/assets/62778100/9778e9b9-27e6-40dc-9239-b70daf7e3772">

**Comment by curryhendry** [comment]
请问小米路由R3D如何挂载？好像不是路径问题，/userdisk/data/

是SMB v1好像

提示：
connection error: read tcp 192.168.31.114:34024->192.168.31.114:445: read: connection reset by peer

**Comment by linyi9102** [comment]
挂载京东云无线宝内置硬盘的方式
![image](https://github.com/alist-org/alist/assets/165453585/a0380dce-f144-4d18-8c06-338096acccc2)

**Comment by Perter1999** [comment]
有大佬帮忙看看吗？我这个提交了还报错，装docker里面的，电脑都能直接访问。
报错：Failed init storage: connection error: invalid transport format
![Snipaste_2024-07-25_07-02-55](https://github.com/user-attachments/assets/60dda75f-35e1-4c81-b0e3-050b6bdd6b4b)

**Reply by songxiaofeng1981** [reply]
分享名称跟你的文件夹名称要一致：比如：
![image](https://github.com/user-attachments/assets/e86265e5-4803-47eb-adb5-29e7f0556c25)
比如我的名称填movie 就可以访问了。不行抓tcp包看看报错

**Comment by dth188** [comment]
我现在的环境是软路由架设alist服务器，局域网内有nas设备，软路由上还拖了一块2t的硬盘，为了达成一站式服，把各种网盘挂载到alist服务器的同时，把软路由的2t硬盘作为本地存储挂载到了Alist服务器，并且把nas以smb协义的方式也挂载到了Alist服务器。外网通过动态域名端口转发可以直接登录内网alist服务器，形成了各种存储一站式服务。由于宽带上传速度只有50兆，在外网访问内网过程中，除了那些通过302代理的网盘以外网实际网速和网盘提供下载网速决定外，其他通过本地代理的存储方式都能跑满50兆。现在要向网友们通报的是在内网环境中使用alist服务器。本人的内网是软路由作为主路由，2.5g设备和1G设备混用。各种网盘使用正常，软路由自带2t存储能跑满速，但是nas的smb存储降速严重。通报的原因可能是套娃结构中协议不兼容产生的降速，比如nas的数据要到软路由中转一下 ，但本身这台nas就连在软路由上，正常情况下也是需要通过软路由中转，但是可以满速。怀疑是在软路由的alist服务器又做了一次本地代理。相当于这个数据在这台软路由上进行了两次转发？

**Comment by HY-Jian** [comment]
华为家庭存储，通过SMB可共享个人文件夹，但还有一个未设密码的家庭共享未显示，通过设置密码也未成功挂载

**Comment by xmlys15** [comment]


**Reply by xmlys15** [reply]
已经成功挂载

**Reply by xmlys15** [reply]
但是用nginx反代alist之后可以打开目录但是打不开视频，必须用内网地址登录才可以打开。

**Reply by SyberRabbit** [reply]
> 但是用nginx反代alist之后可以打开目录但是打不开视频，必须用内网地址登录才可以打开。
> 

alist官网上有在说反代时，要修改的一些server配置

**Comment by annpardo** [comment]
那么怎么通过webdav把软路由连接的储存添加到alist上呢，是了很多次了，不知道那个地方有问题

**Reply by xmlys15** [reply]
一样的道理

**Comment by huamike** [comment]
个人认为这个地方应该是存在一些问题的，我实际测试结果如下：
设置要素：
share name：    #这里填共享名（windows下直接能看到的第一级文件夹的名字）
Root folder paht：     #这里就留个"."

我的测试环境为群晖，做了以下对比测试：
1、超管账号必成
我用群晖的超管账号随便设置哪个文件夹都可以成功。我相信这里很多成功的都是用了自己对应设备的超级管理员账号（或者至少是对整台设备共享文件夹的完全读写账号）

2、普通账号必败
我用了一个只对个别文件夹有完全读写权限的账号，怎么折腾都不行。
然后我用了一个最彻底的方法，直接新建一个一级文件夹（比如share，访问地址为：\\\192.168.0.2\share），然后将这个普通账号对该文件夹设置了读写权限——然后，这样都是失败了。

非常符合这篇帖子里某些反馈人员提到的，”电脑都能打开，偏偏这里用不了“
所以这个smb模式可能有一些过度的权限确认，才造成了这个问题。

**Comment by Allowqent** [comment]
首先要确保使用账户有设置密码，否则无法通过检测机制，share name是填写分享文件夹的名称，如本地路径为：c:/shared,则share name填写shared。address填写的是作为服务器的ip地址+端口号445，如127.0.0.1:445,192.168.x.x:445和域名:445等，Root folder 保持为"."就行，其余的问题不大

**Comment by zymshirley** [comment]
想映射mac的文件夹到alist却提示Failed init storage: response error: The network path cannot be located.

**Comment by maojiaoyu** [comment]
提供一下我的经历，在挂载turenas中的文件夹时，跟文件夹路径填“.”，但是分享名称不能使用pool的名称，要使用数据集的名称。比如我的SMB路径是“/mnt/NAS/Share”，分享名称填写Share即可，填NAS（也就是我的pool名称）没用。
![image](https://github.com/user-attachments/assets/7b2a119d-94a0-428d-9cad-8aee3bf73747)

**Reply by maojiaoyu** [reply]
添加后就可以访问了
![image](https://github.com/user-attachments/assets/be7cc4ee-bf6e-4a8e-a825-0a1470f1a1e4)

**Reply by ywbwtw** [reply]
你说的很对，分享名称应该填IP地址后的第一个文件夹，而不是最后一个文件夹。但是这样会把一级文件夹下面的所有子文件夹都显示出来。我们可以继续缩小范围。分项名称不变的情况下，你可以修改根文件夹路径，在.后面加上\和你想分享的最终文件夹。比如你的就是.\turenas

**Comment by cler1818** [comment]
请问这个IP地址怎么可以改成电脑主机啊，不然换个局域网立马又要重新设置IP  主要是经常带着电脑来回跑
电脑主机名是zhaopeng，设置成IP可以，但是设置主机名称就不行
![1](https://github.com/user-attachments/assets/3bc920c2-87da-4d5d-b50e-5f1511b70440)
![2](https://github.com/user-attachments/assets/8e098ccf-2ab7-4178-9934-65f918bdd9e5)

**Reply by CJY-123456** [reply]
> 请问这个IP地址怎么可以改成电脑主机啊，不然换个局域网立马又要重新设置IP 主要是经常带着电脑来回跑 电脑主机名是zhaopeng，设置成IP可以，但是设置主机名称就不行 ![1](https://private-user-images.githubusercontent.com/28837093/435612397-3bc920c2-87da-4d5d-b50e-5f1511b70440.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY3NDQ0MzYsIm5iZiI6MTc1Njc0NDEzNiwicGF0aCI6Ii8yODgzNzA5My80MzU2MTIzOTctM2JjOTIwYzItODdkYS00ZDVkLWI1MGUtNWYxNTExYjcwNDQwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTAxVDE2Mjg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRjZTk1Zjk4M2VmOTc4MmZlYTdiMzVjNzUxZThmMDdlY2VmNDRhNDE5MzI0OGNlYzczMjdkZGVkOTgxMzJjYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Gce114FFYRdQH0tuUq4o4V9eYln4-pLBnoap61FYmLI) ![2](https://private-user-images.githubusercontent.com/28837093/435612927-8e098ccf-2ab7-4178-9934-65f918bdd9e5.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY3NDQ0MzYsIm5iZiI6MTc1Njc0NDEzNiwicGF0aCI6Ii8yODgzNzA5My80MzU2MTI5MjctOGUwOThjY2YtMmFiNy00MTc4LTk5MzQtNjVmOTE4YmRkOWU1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTAxVDE2Mjg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkZDAxYWI5MmRkZTI3ZmE3OGE1YjFhYTU2MTE4M2U5ZjQwZTk3NjI3NWY1MzEzYTY1NTliZTZhYmVjNmRlZTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.E-FDNol0bUwqVtmXq2Ux3LYVhe7fI0zIQ7yNm0cOIBk)

本机名只有在本地的情况下才能用，相当于127.0.0.1

**Comment by ZerOri** [comment]
smb是不是不支持smb1.0挂载啊？

**Comment by xmi112** [comment]
我的挂载成功了
![image](https://github.com/user-attachments/assets/8106e59b-e4b9-4c39-832c-0a115bcd9dea)
![image](https://github.com/user-attachments/assets/e1ecf125-0514-4b65-b581-c62e329560ff)
真实在共享路径是这样的：\\172.18.143.104\Users\xmi11\Downloads
感觉就是那个小点保留用作根目录，后面继续拼接子目录。

**Comment by xucaibao1979** [comment]
<img width="802" height="918" alt="image" src="https://github.com/user-attachments/assets/93aa40a9-560a-4d54-878c-e41dd83ccdfa" />
<img width="1063" height="352" alt="image" src="https://github.com/user-attachments/assets/402109ec-dffc-426f-9f79-10b7de8ac254" />
使用域服务器共享的文件，添加报错，有遇到的么，怎么解决。

**Comment by virkingz** [comment]
smb挂载最高只有10MB/s的速度

**Reply by CJY-123456** [reply]
我的有50mb/s 还行
